callstack/react-native-webpack-toolkit
# feat: add hermes debugging support

### Summary

- Adds support for inspecting running application inside Hermes VM using Flipper.

### Test plan

1. Run app with Hermes enabled.
2. Open Flipper.
3. Hermes Debugger should show source files and allow to stop on a breakpoints and step over/into, etc.
